web/client/bin/test-lint

```diff
@@ -1,9 +1,22 @@
 #!/bin/bash
 
-pushd /opt/irontec/ivozprovider/web/client
+SCRIPT_DIR=$(dirname "$(realpath "$0")")
+[ -x $(which yarn) ] && YARN=$(which yarn) || YARN=$(which yarnpkg)
+
+echo 
+pushd "${SCRIPT_DIR}"/../
```

Comment:
We use hardcoded paths in all provider scripts, because they're designed to run in development enviroments, where paths and comands are fixed. No docker support is involved here, so this script is always located in the same path and yarn is always yarnkpkg.

Howerver, this is a valid enhancement, but I think It belongs to another PR (update test fixed paths). Also I think we may find a more clear approach for this project.

Reply:
Perfect, I'll roll back changes.